docs/user/filesystem.md

```diff
@@ -121,6 +121,16 @@ to DAOS. It should be run with the credentials of the user, and typically will
 be started and stopped on each compute node as part of the prolog and epilog
 scripts of any resource manager or scheduler in use.
 
+### Core binding and threads.
+
+DFuse will launch one thread per available core by default, although this can be
+changed by the `--thread-count` option. To change the cores that DFuse runs on
+use kernel level tasksets which will bind DFuse to a subset of cores, this can be
+done via the `tasket` or `numactl` programs or similar, if doing this then DFuse
+will again launch one thread per available core by default.  Many metadata
```

Comment:
Minor punctuation nitpick
```suggestion
use kernel level tasksets which will bind DFuse to a subset of cores. This can be
done via the `tasket` or `numactl` programs or similar. If doing this then DFuse
will again launch one thread per available core by default.  Many metadata
```

Reply:
Done.